src/Microsoft.Dnx.Tooling/Building/BuildManager.cs

```diff
@@ -243,18 +249,26 @@ private bool BuildInternal(string projectPath)
                             symbolPackageBuilder.Files.Add(srcFile);
                         }
 
-                        using (var fs = File.Create(nupkg))
-                        {
-                            packageBuilder.Save(fs);
-                            _buildOptions.Reports.Quiet.WriteLine("{0} -> {1}", _currentProject.Name, Path.GetFullPath(nupkg));
-                        }
+                        var packageSuccess = !packDiagnostics.HasErrors();
+                        success &= packageSuccess;
+                        WriteDiagnostics(packDiagnostics);
 
-                        if (symbolPackageBuilder.Files.Any())
+                        // Write the packages
+                        if (packageSuccess)
                         {
-                            using (var fs = File.Create(symbolsNupkg))
+                            using (var fs = File.Create(nupkg))
                             {
-                                symbolPackageBuilder.Save(fs);
-                                _buildOptions.Reports.Quiet.WriteLine("{0} -> {1}", _currentProject.Name, Path.GetFullPath(symbolsNupkg));
+                                packageBuilder.Save(fs);
+                                _buildOptions.Reports.Quiet.WriteLine("{0} -> {1}", _currentProject.Name, Path.GetFullPath(nupkg));
+                            }
+
+                            if (symbolPackageBuilder.Files.Any())
```

Comment:
The inline diff looks weird, split diff is clearer. Regardless, this was in the PR from early on.